lang/semgrep-grammars/src/semgrep-elixir/grammar.js

```diff
@@ -28,6 +28,15 @@ module.exports = grammar(base_grammar, {
       );
     },
 
+    _keyword: ($, previous) => {
+	return choice(
+	    ...previous.members,
+	    $.metavariable_keyword
+      );
+    },
+
+    metavariable_keyword: $ => seq($._semgrep_metavariable, /:\s/),
```

Comment:
Why the required space after : ?
Is it to match what tree-sitter-elixir does for the other keywords?

Reply:
Yes, exactly.